.github/workflows/release.yaml

```diff
@@ -128,6 +130,8 @@ jobs:
       - name: Build & publish release artifacts
         run: make release
         env:
+          GHCR_USERNAME: ${{ github.actor }}
```

Comment:
Approved

Can we sync offline to confirm these values are correctly being assigned at this part in the release process?